examples/azure/main.tf

```diff
@@ -5,6 +5,9 @@ terraform {
 module "azure" {
   source = "git::https://github.com/ComputeCanada/magic_castle.git//azure"
 
+  # If using a pre-created resource group specify it in azure_resource_group
+  # Otherwise leave the variable empty to allow terraform to manage a new resource group
+  azure_resource_group = ""
```

Comment:
Since we define a default value, a type and a description for this variable, and since it is more advanced usage of Azure, I would prefer to not include it in the Azure example.

We should however document it in the azure section of the documentation.